content/pages/meetup/2023/06-nycu.rst

```diff
@@ -0,0 +1,52 @@
+========================================
+Meetup 2023 June in NYCU
+========================================
+
+:date: 2023-06-20 21:30
+:url: meetup/2023/06-nycu
+:save_as: meetup/2023/06-nycu.html
+
+Meetup 2023 June in NYCU
+========================================
+
+* Date: June 7th, 14th, 21st and 28th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+To join the meetup, please bring your laptop. You are also very welcome to bring your project. Please `contact us
+<#contact-us>`__ if you have any questions.
+
+Meetup is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+Agenda
+-------
+* 18:30-19:40 (70 min) Free discussion
+* 19:40-20:05 (25 min) Tech sharing (w/QA)
+* 20:05-20:20 (15 min) Break
```

Comment:
It is unclear why break and off-topic discussion are consecutive.

Reply:
Oh, yes. Perhaps combining the break into "off-topic discussion" will looks fine.